t/util.t

```diff
@@ -0,0 +1,96 @@
+use Test::Nginx::Socket 'no_plan';
```

Comment:
What is this file for? Could you give me some guides?

Reply:
It's test framework in Nginx lua: https://metacpan.org/pod/Test::Nginx::Socket, most of nginx lua module are test by `test::nginx`.